catalyst_voices/lib/pages/widgets/voices_app_bar/actions/finish_account_button.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:flutter/material.dart';
+
+/// A [FinishAccountButton] widget that is used to display a call to action to
+/// complete a pending account registration.
+class FinishAccountButton extends StatelessWidget {
+  final void Function()? onPressed;
```

Comment:
`final VoidCallback? onPressed;`

Reply:
make sure to apply in other places, fyi: another popular defined typedef is `ValueChanged<T>`